smart-contracts/contracts/DigitalaxAuction.sol

```diff
@@ -8,15 +8,48 @@ import "@openzeppelin/contracts/utils/ReentrancyGuard.sol";
 import "./DigitalaxAccessControls.sol";
 import "./DigitalaxGarmentNFT.sol";
 
-// TODO add 721 receiver hook
-
 contract DigitalaxAuction is Context, ReentrancyGuard {
     using SafeMath for uint256;
 
     event AuctionCreated(
         uint256 indexed garmentTokenId
     );
 
+    event UpdateAuctionEndTime(
+        uint256 indexed garmentTokenId,
+        uint256 endTime
+    );
+
+    event UpdateAuctionStartTime(
+        uint256 indexed garmentTokenId,
+        uint256 startTime
+    );
+
+    event UpdateAuctionReservePrice(
+        uint256 indexed garmentTokenId,
+        uint256 reservePrice
+    );
+
+    event UpdateAccessControls(
+        address indexed accessControls
+    );
+
+    event UpdatePlatformFee(
+        uint256 platformFee
+    );
+
+    event UpdatePlatformFeeRecipient(
+        address payable _platformFeeRecipient
```

Comment:
why use _ on this one and not the others?

Reply:
does this need payable here?

Reply:
Payable is to save the mapping, will fix bad variable name to be consistent